openquake/sep/landslide/nowicki_jessee.py

```diff
@@ -0,0 +1,164 @@
+from typing import Union
+import numpy as np
+
+
+LANDCOVER_TABLE={
+    b"14": 0.91,
+    b"20": 0.88,
+    b"30": 0.78,
+    b"40": 0.68,
+    b"50": 0.30,
+    b"60": 1.77,
+    b"70": 1.71,
+    b"90": -1.26,
+    b"100": 1.50,
+    b"110": 0.68,
+    b"120": 1.13,
+    b"130": 0.79,
+    b"140": 1.03,
+    b"150": 0.54,
+    b"160": 2.34,
+    b"170": 1.19,
+    b"180": 1.19,
+    b"190": 0.30,
+    b"200": -0.06,
+    b"220": -0.18,
+    b"230": -1.08,
+    "14": 0.91,
+    "20": 0.88,
+    "30": 0.78,
+    "40": 0.68,
+    "50": 0.30,
+    "60": 1.77,
+    "70": 1.71,
+    "90": -1.26,
+    "100": 1.50,
+    "110": 0.68,
+    "120": 1.13,
+    "130": 0.79,
+    "140": 1.03,
+    "150": 0.54,
+    "160": 2.34,
+    "170": 1.19,
+    "180": 1.19,
+    "190": 0.30,
+    "200": -0.06,
+    "220": -0.18,
+    "230": -1.08
+}
+
+
+LITHOLOGY_TABLE={
+    b"mt": -1.87,
+    b"nd": -0.66,
+    b"pa": -0.78,
+    b"pb": -1.88,
+    b"vi": -1.61,
+    b"py": -1.05,
+    b"sc": -0.95,
+    b"sm": -1.36,
+    b"ss": -1.92,
+    b"su": -1.36,
+    b"va": -1.54,
+    b"vb": -1.50,
+    b"pi": -0.81,
+    "mt": -1.87,
+    "nd": -0.66,
+    "pa": -0.78,
+    "pb": -1.88,
+    "vi": -1.61,
+    "py": -1.05,
+    "sc": -0.95,
+    "sm": -1.36,
+    "ss": -1.92,
+    "su": -1.36,
+    "va": -1.54,
+    "vb": -1.50,
+    "pi": -0.81
+}
+
+
+def sigmoid(x):
```

Comment:
Given this function is also used in the liquefaction classes, we should move it outside and import and reuse the same function everywhere instead of redefining.

Reply:
will be addressed later.